### ISSUE TITLE
feat: add a11y docs to Skeleton

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#31971]
Updating the `aria-label` to not be set by default was a good reminder that we need some a11y docs on how & when to label skeleton loaders, as well as other potential a11y considerations. Specifically they should cover:

- Using `aria-busy` at a higher level than Skeleton if there are a lot of items simultaneously loading and the content is changing quickly enough that you don't want users to start trying to read it until it's done
- If the skeleton is wrapped by a control with a nameable role (e.g. `article`, `group`, `region`, etc.), the label could go on that element
- If there are a bunch of simultaneously loading skeleton loaders and there's any information to differentiate them, they should have unique labels
- Do _not_ fire loading/loaded live regions for each skeleton if there are many skeleton loaders. Loading announcements for the entire page should not be more frequent than once every 5 seconds, and the "loaded" announcement should usually happen only once for all currently-loading content. If content loads as it scrolls into view, it's possible one may not want any live region announcements, as they would just be distractions from the user reading the already-loaded content.
